src/CashierFake.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Laravel\Paddle;
+
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Str;
+use Laravel\Paddle\Events\PaymentSucceeded;
+use Laravel\Paddle\Events\SubscriptionCancelled;
+use Laravel\Paddle\Events\SubscriptionCreated;
+use Laravel\Paddle\Events\SubscriptionPaymentFailed;
+use Laravel\Paddle\Events\SubscriptionPaymentSucceeded;
+use Laravel\Paddle\Events\SubscriptionUpdated;
+
+class CashierFake
+{
+    /**
+     * Initialize the fake instance and fake Cashier's events and API calls.
+     *
+     * @param  array  $endpoints
+     * @param  string|array  $events
+     * @return void
+     */
+    public function __construct(array $endpoints = [], $events = [])
+    {
+        foreach (
+            $endpoints = array_merge([
+                'payment/refund',
+                'subscription/users',
+                'subscription/modifiers',
+                'subscription/modifiers/create',
+                'subscription/modifiers/delete',
```

Comment:
Do we still need these if they're always explicitly given now?

Reply:
Ehhhhh...it depends. 

So the way I intended this `Cashier::fake()` to be used is that a dev could put in a single `Cashier::fake()` and it would automatically just mock any API endpoint and event that Cashier touched. The user doesn't really care about what values are returned from the API, just that the call was successful and Cashier wouldn't throw. If the user cared about what was returned here then they could override the values by providing their own data such as:

```php
Cashier::fake([
  'payment/refund' => [/* ... */]
]);
```

If we remove this list of endpoints from the constructor I think it limits the usefulness of the method because now everytime we're asking a dev to provide the endpoints that need to be mocked every time they want to do a fake. This asks the dev to do two things: 1. Know how Cashier works under the hood by knowing what methods call what endpoints, and 2. Knowing the Paddle API and what values need to be returned (ie, every response _needs_ to have `success: true`

So while we _could_ remove these endpoints and leave it up to the developer using `Cashier::fake()` to do it every time I think that eliminates the usefulness of this method.

I think some of that purpose is getting lost in the back and forth of the code review here so I'm willing to jump on a quick call to get this sorted. You're obviously not in my timezone (GMT-6) but I'm willing to make myself available to just quickly bang through this if you'd like instead. Feel free to DM me or I can ping you on twitter or something....lemme know what's best.

Reply:
You're correct, this is fine 👍